src/domain/event/EventPage.tsx

```diff
@@ -38,10 +38,17 @@ const EventPage = (): ReactElement => {
     string[]
   >([]);
 
-  const { data: eventData, loading } = useEventQuery({
+  const { data: eventData, loading, refetch: refetchEvent } = useEventQuery({
     variables: { id: eventId as string, include: ['keywords', 'location'] },
   });
 
+  // refetch event data (to get updated occurrences) if enrolment was created
+  React.useEffect(() => {
+    if (enrolmentCreated) {
```

Comment:
does this get covered in tests?

Reply:
It is not, maybe I will add simpletest to make sure refetch is called on first render...

Reply:
Now fixed